code/obj/machinery/cashreg.dm

```diff
@@ -147,24 +137,12 @@ TYPEINFO(/obj/machinery/cashreg)
 		user.visible_message("<span class='alert'><b>[src] buzzes.</b> The transaction was cancelled!</span>")
 		src.active_transaction = FALSE
 
-	// This proc should really exist elsewhere.
-	proc/get_ID(obj/item/O)
-		if (istype(O, /obj/item/card/id))
-			return O
-		else if (istype(O, /obj/item/device/pda2))
-			var/obj/item/device/pda2/pda = O
-			return pda.ID_card
-
-	/// If an ID is being held, get what's equipped. Otherwise, if mob M is wearing something in their ID slot, get an ID from that.
-	proc/check_worn_ID(mob/M)
-		var/obj/item/O
-		if (ishuman(M))
-			var/mob/living/carbon/human/owner = M
-			if (istype(M.equipped(), /obj/item/card/id) || istype(M.equipped(), /obj/item/device/pda2))
-				O = M.equipped()
-			else if (owner.wear_id)
-				O = owner.wear_id
-			return O
+	proc/fetch_mob_id(mob/user, var/not_worn = FALSE)
+		if (ishuman(user))
+			var/mob/living/carbon/human/human_user = user
+			return human_user.get_id(not_worn)
+		else
+			return user.get_id()
```

Comment:
Why not pass not_worn here too? For future ID-wearing mobcritter purposes.

Reply:
Or rather why have this proc at all instead of calling `usr.get_id()` :shelterfrog: